src/components/Home/WalletCard.tsx

```diff
@@ -1,13 +1,17 @@
-import { MouseEvent } from 'react'
+import { BG_WALLET_CARD_SVG } from '@/assets'
+import PeanutWalletIcon from '@/assets/icons/small-peanut.png'
 import { Card } from '@/components/0_Bruddle'
-import Image from 'next/image'
-import { motion } from 'framer-motion'
-import classNames from 'classnames'
-import PeanutWalletIcon from '@/assets/icons/peanut-wallet.png'
 import Icon from '@/components/Global/Icon'
-import { shortenAddressLong, printableUsdc } from '@/utils'
-import { IWallet } from '@/interfaces'
-import { useToast } from '@/components/0_Bruddle/Toast'
+import { IWallet, WalletProviderType } from '@/interfaces'
+import { printableUsdc, shortenAddressLong } from '@/utils'
+import classNames from 'classnames'
+import { motion } from 'framer-motion'
+import Image from 'next/image'
+import { useMemo } from 'react'
+import CopyToClipboard from '../Global/CopyToClipboard'
+
+// convert the color map to an array for easy indexing
+const colorArray = ['#90A8ED', '#E99898', '#FFC900', '#98E9AB']
```

Comment:
Q: what are these colors? Where do they come from? Are they going to be used in other places, not just wallet card? If so, they shouldn't be defined locally here, there should be some other place where our primary colors are defined!

Reply:
@Hugo0 fixed this, pushing in next commit, now using already defined colors, missed that initially